public/css/designer.css

```diff
@@ -0,0 +1,215 @@
+html,
+body {
+  height: 100%;
+}
+
```

Comment:
You might want to use sass instead of css

Reply:
I am not sure what you are getting at. You are using `css` here. If you would use `.sass`, which is a more powerful language than `.css`, then it would indeed by transpiled as you mention.

Reply:
I'm not sure if I misunderstand, but the `public` folder contains the results of transpiling the `.sass` and `.js` files in the `resources` folder

Reply:
We indeed use `.sass`, what you see here are compiled style files. Please look into the directory `resources/assets/sass`. Laravel was shipped with a `.gitignore` file which didn't include those directories, and we didn't decide to change it. That's why they are always included in the commits.